### PR TITLE
Add support for pnpm 9.x

### PR DESCRIPTION
**Pull request by co5m0**
Support for pnpm 9.

Tested on my local machine, and it works as expected.

### CODE REVIEW

**Comment by AlejandroAkbal** [comment]
Please merge this if its ready

**Comment by hacknug** [comment]
Default `pnpm` version should probably be updated to latest before merging

**Review by coffee-cup — APPROVED** [review]
thank you!

**Pull request merged**
